src/types/asam/index.ts

```diff
@@ -1,3 +1,5 @@
+/* tslint:disable */
```

Comment:
Why is this necessary? If it's unavoidable, can we disable the exact rule which is causing problems, rather than disabling TSLint entirely?

Reply:
Hi Mathieu, its connected with https://github.com/dvsa/mobile-examiner-alpha/pull/55#discussion_r180983592 (can we continue thread there?)